packages/lesswrong/lib/collections/postViewTimes/collection.ts

```diff
@@ -0,0 +1,25 @@
+import { createCollection } from '../../vulcan-lib';
+import { addUniversalFields, getDefaultResolvers } from '../../collectionUtils'
+import { getDefaultMutations } from '../../vulcan-core/default_mutations';
+import { schema } from './schema';
+import { ensureIndex } from '../../collectionIndexUtils';
+
+export const PostViewTimes = createCollection({
```

Comment:
Please add documentation for what this collection is, how it's used, and why it exists.